libuuu/sdps.cpp

```diff
@@ -81,6 +81,18 @@ struct _ST_HID_CBW
 
 #include "rominfo.h"
 
+static bool is_ivt_header(shared_ptr<DataBuffer> data, size_t off)
+{
+	if (off + sizeof(IvtHeader) > data->size())
+		return false;
```

Comment:
```>= data->size``

Reply:
Assume sizeof(IvtHeader) == 4 and off == 0. This function needs access to the first 4 bytes only, so 4 == 4 is actually ok. I thus don't tink that >= data->size is needed.

Reply:
yes, you are right